### ISSUE TITLE
Joukowsky aerofoils

### ISSUE THREAD

**Issue by markrichardson** [janniklasrose/aerofoil-generator#2]
Hi Jan - no idea if this is useful or relevant (or trivial) but I saw this repo and it occurred to me that ChebPy v0.4+ should be quite easily able to generate aerofoil shapes via this kind of thing: https://en.wikipedia.org/wiki/Joukowsky_transform

**Comment by janniklasrose** [comment]
Hey Mark. Interesting, I didn't know about this. It's certainly a cool application of ChebPy. But I don't really want to put non-system dependencies into this repo, the idea was that the .py's (well, one .py so far) are standalone "binaries". You might notice that I'm not even including numpy or matplotlib here.

**Comment by markrichardson** [comment]
makes sense, and yup purely a general comment rather than a specific suggestion for the repo
